PerDiem/PerDiem/models/TransactionList.h

```diff
@@ -7,12 +7,15 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "DateTools.h"
 
 @interface TransactionList : NSObject
 
 @property (strong, nonatomic) NSArray *transactions;
 @property (strong, nonatomic) NSNumber *sum;
 
-- (id) initWithTransactions: (NSArray*) transactions;
+- (id)initWithTransactions:(NSArray*)transactions;
++ (id)transactionListWithTransactionList:(TransactionList *)transactionList
+                        filterWithPeriod:(DTTimePeriod *)timePeriod;
```

Comment:
I would prefer a public method that's call filterWithPeriod and it returns a new instance of TransactionList containing all the transactions within the period. I think is clearer